iothub_client/src/blob.c

```diff
@@ -328,6 +328,11 @@ BLOB_RESULT Blob_UploadMultipleBlocksFromSasUri(const char* SASURI, IOTHUB_CLIEN
                     LogError("unable to create a HTTPAPIEX_HANDLE");
                     result = BLOB_ERROR;
                 }
+                else if ((timeoutInMilliseconds != 0) && (HTTPAPIEX_SetOption(httpApiExHandle, "timeout", &timeoutInMilliseconds) == HTTPAPIEX_ERROR))
```

Comment:
This should be `OPTION_HTTP_TIMEOUT` from `shared_util_options.h`.  The `TrustedCerts" pre-existing online 336 ideally could get piggy back fixed here to use `OPTION_TRUSTED_CERT`.

Reply:
Fixed!